lib/core/errors.c

```diff
@@ -5,6 +5,9 @@ int __log_level = 3;
 int __enable_syslog = 1;
 const int __max_log_msg_len = 1024;
 
+enum ReturnCode __status_code = CLEAN_EXIT;
+char __err_msg[1024] = "Hello there";
```

Comment:
Was the `"Hello there"` forgotten here?

Reply:
Yep, I just realised that. I've removed it now.